inspirehep/dojson/experiments/fields/bd1xx.py

```diff
@@ -171,10 +171,10 @@ def date_started(self, key, value):
     values = force_force_list(value)
 
     for val in values:
-        if val.get('s'):
-            self['date_started'] = val.get('s')
-        if val.get('t'):
-            self['date_completed'] = val.get('t')
+        self['date_proposed'] = val.get('q')
+        self['date_approved'] = val.get('r')
+        self['date_started'] = val.get('s')
+        self['date_completed'] = val.get('t')
```

Comment:
My bad: my previous review was incorrect! The problem here is that, if you have another `046` which does not contain an `s` subfield, the one you collected before gets overwritten by `None`, which leads to https://travis-ci.org/inspirehep/inspire-next/jobs/149504176#L1333.